pandas/core/apply.py

```diff
@@ -18,8 +18,11 @@
 
 from pandas.core.construction import create_series_with_explicit_dtype
 
+from pandas.core.frame import DataFrame
+from pandas.core.series import Series
+
 if TYPE_CHECKING:
-    from pandas import DataFrame, Series, Index
+    from pandas import Index
```

Comment:
I am not completely sure when and how TYPE CHECKING is true.
I hope to find (by someones advice) another way to convert the reduced partial result without using both series and DF.
and return this to the way it was

Reply:
> I am not completely sure when and how TYPE CHECKING is true.

It's for the `mypy` part of the checks, see [type hints](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#type-hints)

Reply:
Yea so its True just for type checkers..
Like i said, i hope to find a better way than using DF and Series in the code.